### ISSUE TITLE
Staging Review finding: Loading component is not announced for assistive technology

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#129390]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ursa Minor
**Product:** Travel Pay Claims
**Feature:** Complex Claims v1

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.]()
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

Navigating the flow, whenever the screen is in a loading state, the loading message is not announced to assistive technology. This lack of information while the system is performing an action may be confusing for assistive technology users - especially screen reader users, who may be confused why there's nothing on the page they can interact with except for the header and footer. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="508" height="187" alt="loading not announced" src="https://github.com/user-attachments/assets/8e6e3609-370d-425b-a451-4f4fdb5bcf59" />

</details>

## Recommended action

Ensure that the loading state is actively announced to assistive technology through the use of `aria-live` regions. 

https://design.va.gov/components/loading-indicator#accessibility-considerations

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 4.1.3 Status Messages - Level AA](https://www.w3.org/TR/WCAG22/#status-messages)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by charrd49** [comment]
This might be something more appropriate for Platform to fix. UX to check with MHV Accessibility (Steven & Sarah) to determine what is needed.
